custom_components/better_thermostat/climate.py

```diff
@@ -733,9 +736,6 @@ async def _async_trv_changed(self, event):
 		if new_state is None or old_state is None:
 			return
 		
-		# fetch device model from HA if necessary
```

Comment:
shouldn't we make sure that we have the device model fetched, at this stage?

Reply:
Not needed, we load the device model at startup, if it fails we exit with an error

Reply:
okay